examples/with-supabase/components/form-message.tsx

```diff
@@ -12,7 +12,7 @@ export function FormMessage({ message }: { message: Message }) {
         </div>
       )}
       {"error" in message && (
-        <div className="text-destructive-foreground border-l-2 border-destructive-foreground px-4">
+        <div className="text-red-500 border-l-2 border-destructive-foreground px-4">
```

Comment:
colour wasn't working in light mode, so went with red

Reply:
weird, i'll check what's wrong.

shouddn't use hard coded colors